### PR TITLE
Initial draft for PR Guidelines

### PR DESCRIPTION
**Pull request by Sagar2366**
Signed-off-by: Sagar Utekar <sagarutekar2366@gmail.com>

Summary :
This PR is added with initial draft for PR guidelines to follow.

### CODE REVIEW

**Comment by palisadoes** [comment]
These are good general guidelines. 

We should have a GSoC specific section that:

1. Outlines how new issues get created (At the moment we are saying that students cannot open them, but they may have good ideas that should be an issue) It implies that a mentor should be involved, we just need to explain how this is done.
2. Adds the line or something similar "If you are borrowing code, please disclose it. It is fine and sometimes even recommended to borrow code, but we need to know about it to assess your work. If we find out that your pull request contains a lot of code copied from elsewhere, we will close the pull request."

**Comment by jordanliu** [comment]
Hey @Sagar2366, could you also implement the following issue & pr templates in this PR as well: #293 - I think having a comment in the PR to read the guides would be helpful

**Comment by Sagar2366** [comment]
Sure @jordanliu, I'll add templates for issue and pr and other necessary details.

**Review by jordanliu — CHANGES_REQUESTED** [review]

File: PR-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Pull Request Guidelines
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+In order to give everyone a chance to submit a pull request and contribute to Talawa project, we have put restrictions in place. This section outlines the guidelines that should be imposed upon pull requests at Talawa. 
+
+1. Do not start working on any open issue and raise PR unless it is assigned to you.
+2. Pull requests must be based on [open issues](https://github.com/PalisadoesFoundation/talawa/issues) available.
+3. Each contributor may only create one pull request at a time. We have this rule in place due to our limited resources - if everyone was allowed to post multiple pull requests we will not be able to review them properly. It is also better for contibutors because you can focus on creating one quality PR - so spend time making sure it is as good as it can be.
```
[the remark this answers is not in the document]
Comment (palisadoes) [reply PR-guidelines.md:9]: It's unlikely that someone will need to work on multiple PRs. If they are, then at least one PR is suffering in code quality. I vote for keeping it.

[the remark this answers is not in the document]
Comment (jordanliu) [reply PR-guidelines.md:9]: @heyjordn what do you think?

[the remark this answers is not in the document]
Comment (palisadoes) [reply PR-guidelines.md:9]: Also, someone could hold things up by having multiple PRs that they forget about.

[the remark this answers is not in the document]
Comment (jordanliu) [reply PR-guidelines.md:9]: Fair point

**Review by jordanliu — APPROVED** [review]

**Pull request merged**